nova/virt/lxd/driver.py

```diff
@@ -1132,13 +1132,13 @@ def finish_migration(self, context, migration, instance, disk_info,
 
         # Step 3 - Start the network and container
         self.plug_vifs(instance, network_info)
-        self.client.container.get(instance.name).start(wait=True)
+        self.client.containers.get(instance.name).start(wait=True)
 
-    def confirm_migration(self, migration, instance, network_info):
+    def confirm_migration(self, context, migration, instance, network_info):
```

Comment:
Where is `context` used here? Or is that that the callers are passing that it?

Reply:
Does `nova/tests/unit/virt/lxd/test_driver.py` need to be updated to pass `context`?

Reply:
'context' argument is being passed in by the caller, the  LXD driver function here does not use this value. 
You are correct that 'test_driver.py' would need to be updated with a 'context' argument value when calling 'confirm_migration()' to work moving forward.  Would you recommend that I just update the test to pass an empty value for context?

Reply:
I suppose we don't necessarily have to make any changes since we don't actively run the tests. I'm fine either way.

Reply:
From looking at OpenStack nova source code, I agree that it looks like the OpenStack is passing `context` to `confirm_migration` in the LXD case. Interestingly, for Xen and VMware, `context` is not passed, but for other cases, it is, so I agree that should be there.